spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingAdapterRunner.java

```diff
@@ -17,16 +17,15 @@
 package org.springframework.cloud.stream.config;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
 import org.springframework.cloud.stream.adapter.ChannelBindingAdapter;
-import org.springframework.context.annotation.Configuration;
 
 /**
  * @author Dave Syer
  *
  */
-@Configuration
-public class LifecycleConfiguration implements CommandLineRunner {
+public class ChannelBindingAdapterRunner implements ApplicationRunner {
```

Comment:
Yeah, `CommandLineRunner` works fine. I just thought `ApplicationRunner` could be a better fit as this class isn't run via cmdLine directly ;)